src/Components/Web.JS/src/DomWrapper.ts

```diff
@@ -35,3 +42,23 @@ function focusBySelector(selector: string): void {
     element.focus();
   }
 }
+
+function elementExists(identifier : string) : boolean {
```

Comment:
Sorry to be nitpicky, but is it possible to tweak the code style to match the other TypeScript code? For example, this would be:

```suggestion
function elementExists(identifier: string): boolean {
```

I know it's only a minor detail 😄

Reply:
Sure. I will fix in other places too.